source/clear-linux/tutorials/wordpress/wp-install.rst

```diff
@@ -1,43 +1,51 @@
 .. _wp-install:
 
-Create a Clear Linux WordPress server
+Set up WordPress\* on a LAMP web server
 #######################################
 
-This tutorial shows you how to install the WordPress components on your |CL| LAMP server. At the end of :ref:`web-server-install`, you created a WordPress\*-ready database using phpMyAdmin\* and MariaDB\*. Now that the LAMP server is up and running, you can add the WordPress components needed to host a WordPress website on your system.
+This tutorial shows you how to install the WordPress\* components on your |CL| 
```

Comment:
@krisdale  |CL| is correct here; however, `frankly mr. shankly... ` , |CL| is not rendering in test build. Will you please check this page after merge? Please confirm attribution worked. It may be nothing, but we should check.

Reply:
LGTM.